avocado/utils/network/interfaces.py

```diff
@@ -380,3 +380,15 @@ def restore_from_backup(self):
         else:
             raise NWException(
                 "Backup file not available, could not restore file.")
+
+    def interface_present(self):
+        """
+        Check if interface is present
+        rtype: bool
+        """
+        cmd = 'ip link show dev {}'.format(self.name)
+        try:
+            run_command(cmd, self.host, sudo=True)
+            return True
+        except Exception as ex:
+            return False
```

Comment:
I wonder if there is a need to return `True/False` in this method as it is just checking if the interface is available. See the following method as an example:

https://github.com/avocado-framework/avocado/blob/ca0bacdbb4d4a52ae2a36b312d4bac691d5740f5/avocado/utils/network/interfaces.py#L242

Reply:
Okay, I had in mind that we prefer to raise an exception instead of return True/False to be more OO, but I recall it from an old discussion. I don't know to which extent it is still true.

Reply:
Your understanding is right, We should always try to use exceptions for upper layers. But in this case, the method name is a question, and users are expecting true/false.

Reply:
Okay, got it!

Reply:
Do you want me to include `ping_check` to return True/False in this PR?

Reply:
@cforno12, I guess this could be in another PR since will break the API, we need to make sure will not break user cases.